anchore_engine/services/apiext/swagger/swagger.yaml

```diff
@@ -2576,6 +2576,31 @@ paths:
       description: Request a jwt token for subsequent operations, this request is authenticated with normal HTTP auth
       consumes:
         - application/x-www-form-urlencoded
+      parameters:
+        - name: grant_type
+          in: formData
+          required: true
+          description: 'OAuth Grant type for token'
+          type: string
+          enum:
+            - password
+        - name: username
+          in: formData
+          required: true
+          type: string
+          description: 'User to assign OAuth token to'
+        - name: password
+          in: formData
+          required: true
+          type: string
+          description: 'Password for corresponding user'
+        - name: client_id
+          in: formData
+          required: true
+          description: 'The type of client used for the OAuth token'
+          type: string
+          enum:
+            - anonymous
```

Comment:
@dakaneye, does this have to be an enum? The service should just default to `anonymous` if the value is not supplied. I don't think API should fail validation

Reply:
I wasn't sure if that was required for now, the [docs](https://docs.anchore.com/current/docs/engine/usage/api_usage/authentication/#bearer-tokensoauth) seem to suggest it. I will just add it as the default value though 👍

Reply:
Defining and default value for a parameter that's required doesn't do anything. If the client does not supply the param, api validation kicks in and fails the request. Defaults are assumed when a param is optional and the value is not supplied by the client